README.md

```diff
@@ -1,4 +1,6 @@
-# Vault Helm Chart
+# Vault Helm Chart [![CircleCI](https://circleci.com/gh/TakeScoop/vault-helm/tree/master.svg?style=svg)](https://circleci.com/gh/TakeScoop/vault-helm/tree/master)
+
+**Maintainer:** [@ryanwholey](github.com/ryanwholey)
 
```

Comment:
Can you add a note explaining why we forked this and what the upstream needs to implement so that we can stop having to maintain our own fork?

Reply:
good idea!